pkgs/tools/audio/beets/default.nix

```diff
@@ -27,6 +27,17 @@ let
       url = "https://github.com/beetbox/beets/commit/bcc79a5b09225050ce7c88f63dfa56f49f8782a8.patch?full_index=1";
       hash = "sha256-Y2Q5Co3UlDGKuzfxUvdUY3rSMNpsBoDW03ZWZOfzp3Y=";
     })
+    (fetchpatch {
+      # Already on master. TODO: remove when updating to the next release
+      # Issue: https://github.com/beetbox/beets/issues/5527
+      # PR: https://github.com/beetbox/beets/pull/5650
+      name = "fix-im-backend";
+      url = "https://github.com/beetbox/beets/commit/1f938674015ee71431fe9bd97c2214f58473efd2.patch";
+      hash = "sha256-koCYeiUhk1ifo6CptOSu3p7Nz0FFUeiuArTknM/tpVQ=";
+      excludes = [
+        "docs/changelog.rst"
+      ];
+    })
```

Comment:
Adding this at the top, would help making the stable `beets` derivation not change at all, since the order of patches makes the Nix compute a different output hash for a derivation.